boards/FSM/Src/App/App_AcceleratorPedalSignals.c

```diff
@@ -73,10 +90,22 @@ bool App_AcceleratorPedalSignals_HasAppsAndBrakePlausibilityFailure(
                25.0f;
 }
 
+bool App_AcceleratorPedalSignals_IsAppsAndBrakePlausibilityOk(
+    struct FsmWorld *world)
+{
+    struct AcceleratorPedals *papps_and_sapps =
+        App_FsmWorld_GetPappsAndSapps(world);
+
+    return App_AcceleratorPedals_GetPrimaryPedalPercentage(papps_and_sapps) <
+           5.0f;
+}
+
 void App_AcceleratorPedalSignals_AppsAndBrakePlausibilityFailureCallback(
     struct FsmWorld *world)
 {
     struct FsmCanTxInterface *can_tx = App_FsmWorld_GetCanTx(world);
+
+    App_CanTx_SetPeriodicSignal_MAPPED_PEDAL_PERCENTAGE(can_tx, 0.0f);
     App_CanTx_SetPeriodicSignal_PLAUSIBILITY_CHECK_HAS_FAILED(
         can_tx,
```

Comment:
How is this un-set. Should we not have a callback for the exit condition?

Reply:
The mapped pedal percentage and the critical fault is set in `AirOpenStateRunOnTick100Hz` and `AirClosedStateRunOnTick100Hz`. It gets overwritten by the callback triggered from the entry condition. When the callback is no longer being triggered, the pedal percentage should be whatever value we set it to in the state machine